preql/dialect/enums.py

```diff
@@ -33,18 +38,31 @@ def default_engine(self):
             from sqlalchemy import create_engine
 
             return create_engine(r"duckdb:///:memory:", future=True)
+        elif self == Dialects.POSTGRES:
+            logger.warn("WARN: Using experimental postgres dialect. Most functionality will not work.")
+            import importlib
+            spec = importlib.util.find_spec("psycopg2")
+            if spec is None:
+                raise ImportError("postgres driver not installed, installed extra postgres dependencies")
```

Comment:
nit: is there a reason not to just try to import it and let it fail? If we're doing this for a custom message, maybe specify exactly the command to run to install or that it's expecting psycopg2